### PR TITLE
Homework 1 and 2 Solution

### PR DESCRIPTION
**Pull request by norogoth**
This is my solution. Thanks for grading!

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,32 @@ class Program
...
+                Console.Write("How old are you? ");
+                string age = Console.ReadLine();
+
+                Console.Write("How much do you weigh? ");
+                string weight = Console.ReadLine();
+
+                Console.Write("How tall are you? ");
+                string height = Console.ReadLine();
+
+                double bmi = Convert.ToInt32(weight) / Convert.ToInt32(height) ^ 2;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove the unused local variable 'bmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4958692)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Program.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4958692&bid=16385754)

**Comment by cptcrunchy** [comment]
Good job on making your pull request(PR)!
As a side note. Please be more aware of what you title your PR. As mentors we want to have a clear idea of what we are reviewing. This PR would be improved if it was clear what your "solution" was to. Which homework assignment. 

In short: Make meaningful Pull Request / Commit titles. 
😄

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,32 @@ class Program
...
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            int x = 2;
+            while (x > 0)
+            {
+                //Thank you for reviewing my code. You guys are amazing.
+
+                Console.Write("Hello. What is your name? ");
+                string name = Console.ReadLine();
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Prefer to use var

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -6,7 +6,32 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            int x = 2;
+            while (x > 0)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: for loop would be more simple, but this works too.

**Comment by norogoth** [comment]
ヽ(・∀・)ﾉ
Thank you.

File: Src/BootCamp.Chapter/Program.cs (hunk 2 above)
Comment (ellmaster) [reply Src/BootCamp.Chapter/Program.cs]: I kind of disagree. This is a bit opinion based IMO. However it is clearly possible here.

**Pull request closed without merging**